apps/mainsite/middleware.py

```diff
@@ -46,6 +47,7 @@ def __call__(self, request):
 
     def process_exception(self, request, exception):
         logger.exception(exception)
+        traceback.print_exc()
```

Comment:
Where does the traceback get printed? Shouldn't it be part of the message sent to the logger?

Reply:
The stack-trace goes to sys.err. I have added it because when developing locally I did not see any stack-traces anymore. I have checked the logs and it does not seem to pollute the logs.